Frida/Script.swift

```diff
@@ -204,6 +204,43 @@ public class Script: NSObject, NSCopying {
             }
         }
 
+        if let dict = message as? [String: Any],
+            let type = dict["type"] as? String,
```

Comment:
This as well, the `type` field is always there and is either `send`, `error`, or `log`.